examples/medplum-patient-intake-demo/src/bots/core/intake-form.ts

```diff
@@ -105,6 +98,9 @@ export async function handler(event: BotEvent<QuestionnaireResponse>, medplum: M
     { valueDateTime: convertDateToDateTime(answers['estimated-delivery-date'].valueDate) }
   );
 
+  // Handle coverage
+
+  const questionnaire = await medplum.readResource('Questionnaire', (response.questionnaire as string).split('/')[1]);
```

Comment:
We have helper function

```suggestion
  const questionnaire = await medplum.readResource('Questionnaire', resolveId(response.questionnaire));
```

Reply:
@filipeximenes you're right, my mistake. There's a bit of an edge case for why `response.questionnaire`.

Actually, I think the preferred method would be 
```ts
const questionnaire = await medplum.readReference({ reference: response.questionnaire });
```